https://bugs.gentoo.org/829183
https://github.com/greenbone/gvm-libs/commit/de63a18fd2263b89bed70c00f7232939b0744002

From de63a18fd2263b89bed70c00f7232939b0744002 Mon Sep 17 00:00:00 2001
From: hubblexplorer <81538364+hubblexplorer@users.noreply.github.com>
Date: Mon, 20 Feb 2023 06:11:00 +0000
Subject: [PATCH] Fix: Resolve error on compile (#741)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

* Resolve error on compile

Fix for error " `pcap-config --libs` contains leading whitespace. "

* Approved changes

Co-authored-by: Juan José Nicola <jjnicola@gmail.com>

---------

Co-authored-by: Juan José Nicola <jjnicola@gmail.com>
Co-authored-by: Juan José Nicola <juan.nicola@greenbone.net>
--- a/boreas/CMakeLists.txt
+++ b/boreas/CMakeLists.txt
@@ -62,6 +62,7 @@ if (PCAP_CONFIG)
   execute_process (COMMAND pcap-config --libs
     OUTPUT_VARIABLE PCAP_LDFLAGS
     OUTPUT_STRIP_TRAILING_WHITESPACE)
+  string(STRIP "${PCAP_LDFLAGS}" PCAP_LDFLAGS)
   execute_process (COMMAND pcap-config --cflags
     OUTPUT_VARIABLE PCAP_CFLAGS
     OUTPUT_STRIP_TRAILING_WHITESPACE)

